### ISSUE TITLE
Site Manager: Need ability to delete a site we are the owner of

### ISSUE THREAD

**Issue by onaseef** [onaseef/yomobi#161]
There should be a way to delete a site that I am the owner of. If I hit my 10 site limit, I'm kinda stuck if I can't hand over ownership of the site to someone else.

**Comment by gilbert** [comment]
What should happen when sites are deleted? Should their url become available again?

**Comment by onaseef** [comment]
Yes.  When a site is deleted by the owner, the URL should become available again.

**Comment by gilbert** [comment]
In that case the public database associated with that site should be either moved or destroyed. Do we want to actually destroy data? Or just mark as deleted?

**Comment by onaseef** [comment]
We might as well just delete the account data.

**Comment by gilbert** [comment]
Ok, implementing cascade deletes on private data, deleting data on the public db, and implementing the ui would take about 5 hours total.

**Comment by onaseef** [comment]
What is cascading delete?

**Comment by gilbert** [comment]
It's deleting data models that depend on others. For example, to properly delete a company, you should delete its followers, uploaded photos, etc.
